bench_manager/bench_manager/doctype/site/site.py

```diff
@@ -207,8 +208,9 @@ def get_installable_apps(doctype, docname):
 	with open(app_list_file, "r") as f:
 		apps = f.read().split("\n")
```

Comment:
This function is really destruction where we using this?

Reply:
this return list for pop button install the app
list contains apps not installed on the site, and the user can install one.